### PR TITLE
Add carousel component for the landing page

### PR DESCRIPTION
**Pull request by xitter**
## Description
Added a carousel component that shows courses powered by GraphQL. Unit tests have been added and keyboard accessibility has been taken care of.

Icon and subtitle can be modified as per the review discussion. I would recommend having icon and subtitle in the GraphQL itself. 

## Screenshots

Screenshot 2019-10-05 at 2 36 40 PM

## Related Issues
https://github.com/wayfair/awesome-learning/issues/83

### CODE REVIEW

File: src/components/Carousel/Carousel.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  display: flex;
+  align-items: center;
+  .Carousel-listWrapper {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    overflow-x: hidden;
+  }
+  .Carousel-list {
+    display: flex;
```
Comment (SheaBelsky) [inline src/components/Carousel/Carousel.scss]: Do `Carousel-list` and `Carousel-listWrapper` not inherit `display: flex` from their parents, or do you need to manually specify it here?

File: src/components/Carousel/Carousel.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    flex-direction: row;
+    position: absolute;
```
Comment (SheaBelsky) [inline src/components/Carousel/Carousel.scss]: Could you describe what `position: absolute` is doing here? Just for clarity's sake.

File: src/components/Carousel/Carousel.scss (hunk 1 above)
Comment (xitter) [reply src/components/Carousel/Carousel.scss]: No, they won't be inheriting by default. It has to be specified manually.

File: src/components/Carousel/Carousel.scss (hunk 2 above)
Comment (xitter) [reply src/components/Carousel/Carousel.scss]: Absolute position is to make carousel animate on basis the left property.

**Comment by Cooperbuilt** [comment]
@SheaBelsky feel free to review as you normally would. My only questions before you are free to complete the review would be- 
1. How does it look on responsive widths (small mobile through desktop)
2. Is it accessible

**Comment by xitter** [comment]
@cooperbuilt hero cards are not displayed on mobile screen, thus I have not considered that.
On the accessibility, enter keydown on left right has been handled and alt text has been added.

**Pull request merged**